src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -103,8 +103,7 @@ public TelegramBotClient(string token, HttpClient? httpClient = null)
             IRequest<TResult> request,
             CancellationToken cancellationToken = default)
         {
-            var url = _baseRequestUrl + request.MethodName;
-
+            var url = new Uri($"{_baseRequestUrl}{request.MethodName}", UriKind.Absolute);
```

Comment:
what about avoiding interpolation by storing _baseRequestUrl as Uri and use `Uri(Uri,String) constructor instead?

```csharp
var url = Uri(_baseRequestUrl, request.MethodName);
```

https://docs.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=netcore-3.1#System_Uri__ctor_System_Uri_System_String_